app/javascript/mastodon/reducers/compose.js

```diff
@@ -223,7 +223,7 @@ const hydrate = (state, hydratedState) => {
   state = clearAll(state.merge(hydratedState));
 
   if (hydratedState.has('text')) {
```

Comment:
Sorry, it should be `hydratedState.get('text')`: indeed, the normal interface has a `text` key with an empty value… which means the change as-is would make the single-column interface back to having auto-focus on the form despite the intent

Reply:
Sorry, I missed that...